packages/cli/src/DebugInfo.ts

```diff
@@ -41,6 +51,10 @@ export class DebugInfo implements Command {
       return this.help()
     }
 
+    // Loads the `.env` file if it exists
+    // and prints a line to stdout to inform the users
+    loadEnvFile(undefined, true)
```

Comment:
Should we rename that function maybe to transport all that is in that comment?
`loadEnvFileIfExistsAndPrintToStdOut()`

Reply:
I destructured the params instead